packages/suite-desktop-ui/src/support/DesktopUpdater/Downloading.tsx

```diff
@@ -46,7 +46,7 @@ export const Downloading = ({ hideWindow, progress }: DownloadingProps) => {
     }, [step]);
 
     return (
-        <NewModal onCancel={hideWindow}>
+        <NewModal onCancel={hideWindow} isBackdropCancelable={false}>
```

Comment:
Sorry, I was unclear about this. I would either:
A) Do nothing and keep the modal as default.
B) Disable all options to close the modal (X button, ESC, backdrop click) and add a "Download in the background" button.

This feels like an incomplete solution.